### ISSUE TITLE
Transactions appear and disappear, primary + secondary addresses and staking pools empty when transactions disappear 13.2.0 Win10

### ISSUE THREAD

**Issue by gab81** [EquilibriaCC/Equilibria#117]
So, we undertand that the transactions appear and disappear, but i also noticed that when they "disappear", then if you click on "receiving addresses", it's all blank, there's no primary nor secondary address, that's pretty bad.

When you first load the program, then transactions appear for a little while so there's a small window of time when you can grab your receiving address. Then it's gone and there's no way to see it, unless you restart the app, obviously not great.

Also there's nothing under staking pools when that happens, all empty. Hope this can be fixed asap. thank you

**To sum it up:**

System: Win10
Latest release 13.2.0
Local Node (full sync)
Balance Shows OK
Transactions Appear and disappear
Receiving Addresses Only appear when transactions appear
Staking Pools: Only appear when transactions appear

**Actions taken**: restart the app, same behaviour repeats itself, appearing transactions, all OK, then gone and all is gone.

thank you

**Comment by AdminXeq** [comment]
Use equilibria-wallet repository for this issues.
Working on it actually on branch testing, when ready will be merged and new release tagged.
https://github.com/EquilibriaCC/equilibria-wallet/commit/9591f7d29b2dfe6aabd006fe3f10dc060ceac53a
